homeassistant/components/blebox/switch.py

```diff
@@ -0,0 +1,35 @@
+"""BleBox switch implementation."""
+from homeassistant.components.switch import SwitchDevice
+
+from . import BleBoxEntity, create_blebox_entities
+from .const import BLEBOX_TO_HASS_DEVICE_CLASSES, DOMAIN, PRODUCT
+
+
+async def async_setup_entry(hass, config_entry, async_add):
+    """Set up a BleBox switch entity."""
+
+    product = hass.data[DOMAIN][config_entry.entry_id][PRODUCT]
+    create_blebox_entities(product, async_add, BleBoxSwitchEntity, "switches")
+    return True
+
+
+class BleBoxSwitchEntity(BleBoxEntity, SwitchDevice):
```

Comment:
We should inherit from `SwitchEntity` now instead.

Reply:
Fixed in upcoming https://github.com/home-assistant/core/pull/35552